Tasks/Common/utility-common/package-lock.json

```diff
@@ -44,7 +44,7 @@
     "semver": {
       "version": "5.5.0",
       "resolved": "https://registry.npmjs.org/semver/-/semver-5.5.0.tgz",
-      "integrity": "sha512-4SJ3dm0WAwWy/NVeioZh5AntkdJoWKxHxcmyP622fOkgHa4z3R0TdBJICINyaSDE6uNwVc8gZr+ZinwZAH4xIA=="
```

Comment:
this shouldn't be changed unless there is a change in package.json

Reply:
As discussed offline we should add this one.